CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_tiktok_ads v0.6.0
+[PR #20](https://github.com/fivetran/dbt_tiktok_ads/pull/20) includes the following **BREAKING CHANGE** updates:
+
+# Feature Updates: Conversion Support
+- We have added the following source fields to each `tiktok_ads` end model:
+  - `real_time_conversion`: Number of times your ad resulted in the optimization event you selected.
+  - `total_purchase_value`: The total value of purchase events that occurred in your app that were recorded by your measurement partner.
+  - `total_sales_lead_value`: The monetary worth or potential value assigned to a lead generated through ads.
+  - `total_conversion_value`: The accumulated value of `total_purchase_value` and `total_sales_lead_value`.
+
+- In the event that you were already passing the above fields in via our [passthrough columns](https://github.com/fivetran/dbt_tiktok_ads?tab=readme-ov-file#passing-through-additional-metrics), the package will dynamically avoid "duplicate column" errors.
+> The above new field additions are **breaking changes**  for users who were not already bringing in conversion fields via passthrough columns.
+
+## Under the Hood
+- Created `tiktok_ads_persist_pass_through_columns` macro to ensure that the new conversion fields are backwards compatible with users who have already included them via passthrough fields.
+- Updated `conversion` to be an integer rather than a numeric data type in the `tiktok_ads_source` package, as is the expected behavior of the field. To reflect that data type change, **you will need to do a `--full-refresh`. This is a breaking change.**
```

Comment:
Will a full refresh really be required? We don't leverage any incremental logic in this package so I don't believe this needs to be called out.

Reply:
Ah good point, will remove